### PR TITLE
Add ApplicationCache from GoogleDev Tools

### PR DESCRIPTION
**Pull request by dratler**
Build ApplicationCache from Google Dev Tools
Add model
Add test
#6667

- [X ] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="GoogleStyle" />
```
Comment (shs96c) [inline .idea/codeStyles/codeStyleConfig.xml]: I'm not quite sure how this managed to sneak into the PR....

Comment (dratler) [reply .idea/codeStyles/codeStyleConfig.xml]: You are truly right sorry about that

**Comment by shs96c** [comment]
There appear to be unrelated changes in this PR. I'm not sure how the changes to the project configs or the python classes fit in, and the reformatting of `NetworkInterceptor` is confusing. Would it be possible to just focus this on adding the ApplicationCache CDP commands, please?

**Comment by dratler** [comment]
Hi @shs96c ,
I have done `git pull upstream master` to merge the code with current code to reduce conflicts.
I haven't add any thing else

**Comment by shs96c** [comment]
Thanks for the PR. I've merged it as ef225e3a4ead82bae163e6ab21d49986bc12e0bc

**Pull request closed without merging**